### PR TITLE
Data augmentations

### PR DESCRIPTION
**Pull request by marco-bazzani**

### CODE REVIEW

**Review by Sean1572 — DISMISSED** [review]
LTGM

File: classification/augmentations.py [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+def add_mixup(mixup: Mixup, sequential: torch.nn.Sequential, idx:int) -> Callable:
+    """
+        sequential: Object containing all other data augmentations to
+        be performed
+        idx: Index at which to perform mixup
+
+    Returns: Function which applies all other data augmentations
+    as well as mixup, with order specified by idx
+    """
+    def helper(
```
Comment (benjamin-cates) [inline classification/augmentations.py]: This nested function is causing an error when using torch multiprocessing

**Review by benjamin-cates — CHANGES_REQUESTED** [review]

**Review by benjamin-cates — APPROVED** [review]
Works on my local machine with and without extra jobs

**Comment by benjamin-cates** [comment]
LTGM

**Pull request merged**